easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2015a.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'Octave'
+version = '4.0.0'
+
+homepage = 'http://www.gnu.org/software/octave/'
+description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('PCRE', '8.36'),
```

Comment:
with yes please I mean: yes, please update your PR by bumping this version yourself